### PR TITLE
Feature profile additions

### LINKED ISSUE #34: Profile: User Image
As a user in profile view,
* I would like to add a profile picture
* Until I've uploaded my photo, I'd like an Avatar to be in it's place

### LINKED ISSUE #35: Profile: Personal Description
As a user in profile view,
* I'd like to set a personal description

### LINKED ISSUE #36: Profile: User Email
As a user in profile view,
* I'd like to edit my email address

### LINKED ISSUE #37: Profile: User Shipping Address
As a user in profile view,
* I'd like to edit my shipping address

### LINKED ISSUE #38: Subscriptions: Crate product history
As a user in profile view,
* I'd like to see a history of products that have been delivered and what was kept

### LINKED ISSUE #39: Subscriptions: Next delivery
As a user in profile view,
* I would like to see when my next delivery is coming
* I would also like to have the ability to adjust the date for when I am available

### LINKED ISSUE #43: Environment: Redux DevTools Setup
As a dev, 
I should have the react-redux devTools set up to view the global store, components connected to the store, 
and what actions are being dispatched to reducers as I interact with the UI,
so that I can track the state/update of data and test it when adding new feature code.

### LINKED ISSUE #45: ProfileEdit/Shipping Address Inputs
As a dev, 
on the user EditProfile form, 
I should have text field inputs for:
- Street address
- City
- Zip

And select dropdown for:
- State 

so that the input form fields match the data fields in the table.

### LINKED ISSUE #47: Fix: Profile/Upload user image
As a user, 
when I upload my picture in the profile edit form, 
I should be returned to the profile page
and see my new uploaded image.

### PR DESCRIPTION
**Pull request by lo-la-do-li**
Web directory contains UI/UX additions for the profile page view, edit form, and display crates

## Description
The user is now able to view their profile information on the main Profile page; Click an edit icon that navigates/routes them to a form view/component, edit image, email, street address, city, state, zip, and the user's preferred image; Save the changes made, and view the new information on the Profile page; and finally, the user can view their upcoming deliveries in a table positioned next to their profile/user information card. 
### Features in web/src/...
- modules/user/Profile.js: User info grid/card display added (start: line 49); Deliveries component displayed (line 130)
- modules/user/Deliveries.js: Created Deliveries component to display table including the following data: crate, deliveryDate, edit icon(pending modal functionality), and order#. (start: line 20)
- - calls on pre-existing `getListByUser` action from /subscription/api/actions.js to display subscription info for deliveries
- modules/user/ProfileEdit.js: Form to edit profile info; Route `path = "/user/edit_profile"` added in setup/routes/user.js
- modules/user/api/actions.js: `update(userDetails)` function/action added (start: line 134)
- modules/user/api/state.js: Reducer `UPDATE_USER_DETAILS` switch case added to state (start: line 29)
### Fixes in web/src/...
- modules/user/Profile.js: User uploaded image now displays after edit form submission (fix: line 58)
- - Profile info now separates shipping address into separate fields for streetAddress, city, state, zip
- modules/user/ProfileEdit.js: Form now displays separate input fields and dropdown for shipping address rather than just one string 'shippingAddress'
## Motivation and Context
This PR allows for new data to be displayed and manipulated in the UI, consistent with new data migrations and mutations created in the API.

## How Has This Been Tested?
The UI/UX was tested informally using React and React-Redux devtools to check that state is updating appropriately and that changes to data are rendered correctly on the UI.

## Screenshots (if appropriate):
User Profile Pre-Edit with Deliveries
Screen Shot 2021-02-12 at 9 20 37 AM
Profile Post-Edit

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] I have updated the README to reflect any changes.

### CODE REVIEW

**Review by JoseLopez235 — APPROVED** [review]

File: code/web/src/modules/pages/Onboarding.js [hunk 1]
```diff
@@ -1,119 +1,183 @@
...
+import React, { PureComponent } from "react";
 
 // UI Imports
-import { Grid, GridCell } from '../../ui/grid'
-import { H1, H6 } from '../../ui/typography'
-import Modal from '../../ui/modal/Modal'
-import Button from '../../ui/button'
-import { white } from '../../ui/common/colors'
-import { textLevel1 } from '../../ui/common/shadows'
+import { Grid, GridCell } from "../../ui/grid";
```
Comment (cameronRomo) [inline code/web/src/modules/pages/Onboarding.js:5]: Love the idea of recycling components for use in other components!

File: code/web/src/modules/pages/Onboarding.js [hunk 2]
```diff
@@ -1,119 +1,183 @@
...
-    this.toggleVisible(false)
-  }
+    this.toggleVisible(false);
+  };
 
   render() {
     const steps = [
       /* 1. Welcome to Crate */
-      <Grid alignCenter={true} style={{ height: '100%', textAlign: 'center', color: white }}>
+      <Grid
```
Comment (cameronRomo) [inline code/web/src/modules/pages/Onboarding.js:54]: Very nice! Super modular and easy to read.

File: code/web/src/modules/subscription/api/actions.js [hunk 3]
```diff
@@ -1,143 +1,179 @@
...
-    })
+      isLoading,
+    });
 
-    return axios.post(routeApi, query({
-      operation: 'subscriptions',
-      fields: ['id', 'user { name, email }', 'crate { id, name, description }', 'createdAt']
-    }))
-      .then(response => {
+    return axios
```
Comment (cameronRomo) [inline code/web/src/modules/subscription/api/actions.js:34]: Nice post request! I kinda like axios. Really helps streamline these requests.

**Review by cameronRomo — APPROVED** [review]
Overall this looks great! Really happy that we went with recycling components, as it made things come together quite quickly.

**Pull request merged**